aem65/src/main/java/com/cognifide/qa/bb/aem65/tests/pageobjects/corecomponents/teaser/TeaserComponent.java

```diff
@@ -0,0 +1,21 @@
+package com.cognifide.qa.bb.aem65.tests.pageobjects.corecomponents.teaser;
+
+import com.cognifide.qa.bb.qualifier.PageObjectInterface;
+
+@PageObjectInterface
+public interface TeaserComponent {

```

Comment:
See the comment for GuiceModule - no Java interface required

Reply:
done